### PR TITLE
feat: Expose mapped results query for public use.

### PR DESCRIPTION
**Pull request by Isakdl**
# Changes

Exposes the mappedResultQuery method for public consumption.

Closes: https://github.com/serverpod/serverpod/issues/1410

## Pre-launch Checklist

- [x] I read the [Contribute](https://docs.serverpod.dev/contribute) page and followed the process outlined there for submitting PRs.
- [x] This update contains only one single feature or bug fix and nothing else. (If you are submitting multiple fixes, please make multiple PRs.)
- [x] I read and followed the [Dart Style Guide](https://dart.dev/guides/language/effective-dart/style) and formatted the code with [dart format](https://dart.dev/tools/dart-format).
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`), and made sure that the documentation follows the same style as other Serverpod documentation. I checked spelling and grammar.
- [ ] I added new tests to check the change I am making.
- [x] All existing and new tests are passing.
- [x] Any breaking changes are documented below.

### CODE REVIEW

File: packages/serverpod/lib/src/database/database.dart [hunk 1]
```diff
@@ -192,6 +192,24 @@ class Database {
     );
   }
 
+  /// Executes a single SQL query. A [List] of rows represented of a [Map] with
+  /// the table name and value is another [Map] with the keys as column names and
+  /// the value as the contents of the column.
+  /// You are responsible to sanitize the query to avoid SQL injection.
+  Future<List<Map<String, Map<String, dynamic>>>> mappedResultsQueryDangerously(
```
Comment (SandPod) [inline packages/serverpod/lib/src/database/database.dart:199]: Suggestion: By using dangerously as a prefix it helps group the methods together and marks the most important aspect of them first.

```suggestion
  Future<List<Map<String, Map<String, dynamic>>>> dangerouslyQueryMappedResults(
```

Comment (SandPod) [reply packages/serverpod/lib/src/database/database.dart:199]: I would suggest reflecting this change for the other dangerously method. `queryDangerously`. -> `dangerouslyQuery`

**Review by SandPod — COMMENTED** [review]
Nice quick work! 👍

File: packages/serverpod/lib/src/database/database.dart (hunk 1 above)
Comment (Isakdl) [reply packages/serverpod/lib/src/database/database.dart:199]: alright!

**Review by SandPod — APPROVED** [review]
LGTM! Could maybe be good to have a smoketest for these methods.

**Pull request merged**